src/script/components/MessagesList/Message/ContentMessage/asset/TextMessageRenderer.tsx

```diff
@@ -164,6 +167,7 @@ export const TextMessageRenderer: FC<TextMessageRendererProps> = ({
           containerRef.current = element;
         }
       }}
+      key={`${editedTimestamp}:${text}`}
```

Comment:
This is important: `key` prop change => react knows it's dealing with new element, so the ref changes and it can recalculate message dimentions. (no need to add extra dependencies to deps array)

Reply:
Brilliant 👌

Reply:
From now we'll also store element's reference in state (it changes only with `key` attribute change). Adding reference to deps array now makes it do it's all recalculations whenever element updates (whenever it's edited). WDYT?

Reply:
makes a lot of sense 👌